pkg/config/config.go

```diff
@@ -37,37 +37,37 @@ var (
 )
 
 type ClusterConfig struct {
-	URL string `json:"url"`
-
-	ClusterCIDR          string `json:"clusterCIDR"`
-	ServiceCIDR          string `json:"serviceCIDR"`
-	ServiceNodePortRange string `json:"serviceNodePortRange"`
-	DNS                  string `json:"dns"`
-	Domain               string `json:"domain"`
-	MTU                  string `json:"mtu"`
+	// URL string `json:"url"`
+
+	// ClusterCIDR          string `json:"clusterCIDR"`
+	// ServiceCIDR          string `json:"serviceCIDR"`
+	// ServiceNodePortRange string `json:"serviceNodePortRange"`
+	// DNS string `json:"dns"`
+	// Domain               string `json:"domain"`
+	// MTU string `json:"mtu"`
 }
 
 type DebugConfig struct {
-	Pprof bool `json:"pprof"`
+	// Pprof bool `json:"pprof"`
 }
 
 type MicroshiftConfig struct {
-	ConfigFile string `json:"configFile"`
-	DataDir    string `json:"dataDir"`
-
-	AuditLogDir string `json:"auditLogDir"`
-	LogVLevel   int    `json:"logVLevel"`
-
-	Roles []string `json:"roles"`
-
-	NodeName string `json:"nodeName"`
-	NodeIP   string `json:"nodeIP"`
-
-	Cluster ClusterConfig `json:"cluster"`
-
-	Manifests []string `json:"manifests"`
-
-	Debug DebugConfig `json:"debug"`
+	// ConfigFile string `json:"configFile"`
+	// DataDir    string `json:"dataDir"`
+	//
+	// AuditLogDir string `json:"auditLogDir"`
+	// LogVLevel   int    `json:"logVLevel"`
```

Comment:
How else would we configure the verbosity level?

Reply:
Would it be acceptable to use a default value that will be sufficient for most use cases such as level 3 or 4?

Reply:
We need to have a top-level log-level parameter for convenience purposes. In OCP we usually have those defined at the operator level, but in the case of MicroShift since we don't have any operator, I think such an option should be handled at the MicroShift level directly.

> Would it be acceptable to use a default value that will be sufficient for most use cases such as level 3 or 4?

I don't think so unless you are debugging a verbosity level of 0 is enough for the users. Also in OCP the default is 0, so I think we should stick to that.

With regards to the actual name of this option as well as the values it can take, I think we should reuse the existing mechanism we have for OCP operators to keep it consistent with the existing products: https://github.com/openshift/api/blob/622889ac07cf3dd8787679af0d3176bf292b3185/operator/v1/types.go#L55-L62

Reply:
The log level is going to need to be configurable in MicroShift.